es-design-system/pages/molecules/es-accordion.vue

```diff
@@ -353,11 +349,15 @@ export default {
                         EsAccordionList to signify that this item should start out expanded.
                     `,
                 },
+                {
+                    name: 'headingTag',
+                    description: `
+                        Defaults to h3. The tag used can be customized as needed.
+                        For example, if the accordion is used within a section that is already under an <h3>, it may need to be an h4.
```

Comment:
I'm seeing a line length warning on this line, probably just need to line break mid-way through.